README.md

```diff
@@ -213,3 +216,53 @@ Do you want to contribute to this project? Please take a look at our [contributi
 
 <img src="https://cdn.stackbuilders.com/media/images/Sb-supports.original.png" alt="Stack Builders" width="50%"></img>
 [Check out our libraries](https://github.com/stackbuilders/) | [Join our team](https://www.stackbuilders.com/join-us/)
+
+## Contributors ✨
```

Comment:
Can we put this section before the Stack Builders' signature image to ensure consistency around our OSS repos? 🙂